.gitignore

```diff
@@ -142,4 +142,7 @@ spark_warehouse
 sqlglot/_version.py
 
 # Emacs files
-*~
\ No newline at end of file
+*~
+
+# Auto-generated files
+sqlglotrs/src/token_type.rs
```

Comment:
IMO we should just commit the generated file and add a unit test that verifies the entries are the same across python and rust.

Reply:
What’s the benefit of this approach?

Reply:
Feels like the existing approach is a more fool proof, doesn't require a manual intervention on the developer side and/or a know-how on how to update the source file for the test to pass.

Reply:
That's true.
I think the main benefit is the browse-ability of the repo. It can be annoying trying to trace through some unfamiliar codebase when it has generated code.

But I don't know if that's more valuable than developer experience. So nvm.